databases/postgresql.md

```diff
@@ -0,0 +1,259 @@
+# PostgreSQL
+
+[PostgreSQL](http://www.postgresql.org/) is an object-relational database management system (ORDBMS).
+
+The connection between PostgreSQL and Ruby happens via the [`pg`](https://rubygems.org/gems/pg) gem.
+
+## Installation
+
+[Install PostgreSQL](http://www.postgresql.org/download/)
+
+```
+gem install pg
+```
+
+**ActiveRecord or DataMapper**. You need only one, but we'll show both.
```

Comment:
Sounds like a good idea. The post is too long to follow and they can just choose their preferred ORM from http://recipes.sinatrarb.com/p/databases?#article . Will update it accordingly